### PR TITLE
partial save and debounce

### PR DESCRIPTION
**Pull request by squallstar**

### CODE REVIEW

**Comment by squallstar** [comment]
@hcarneiro for you to review

File: js/interface.js [hunk 1]
```diff
@@ -30,13 +31,25 @@ $('form').submit(function (event) {
...
+      Fliplet.Widget.complete();
+      window.location.reload();
+    } else {
+      Fliplet.Studio.emit('reload-widget-instance', widgetId);
+    }
   });
-});
+}
+
+$('#primaryButtonLabel').keyup($.debounce(function() {
```
Comment (tonytlwu) [inline js/interface.js]: jQuery has a `.debounce()` API? YASSSS <3

Comment (tonytlwu) [inline js/interface.js]: I would include other events. Typically I'd also include: `mousedown`, `paste`, `input` and `change`

**Comment by squallstar** [comment]
@tonytlwu unfortunately there's no debounce API natively, I've added this tiny plugin https://github.com/Fliplet/fliplet-widget-primary-button/blob/0698e6fef696205d7bab5f0183c8a9608a5c24e1/vendor/jquery.debounce.js

**Comment by tonytlwu** [comment]
Oh well, it's good nonetheless 👍

**Comment by tonytlwu** [comment]
All good with me

**Pull request merged**